internal/controllers/gateway/gateway_utils_test.go

```diff
@@ -525,3 +526,127 @@ func TestRouteAcceptedByGateways(t *testing.T) {
 		})
 	}
 }
+
+func Test_isEqualListenersStatus(t *testing.T) {
+	type args struct {
+		a []gatewayapi.ListenerStatus
+		b []gatewayapi.ListenerStatus
+	}
+
+	lastTransitionTime := metav1.Now()
+	lastTransitionTime2 := metav1.NewTime(lastTransitionTime.Add(time.Second))
+
+	tests := []struct {
+		name string
+		args args
+		want bool
+	}{
+		{
+			name: "",
```

Comment:
Any chance to specify this?

Reply:
Very good suggestion, I also took care of the case when both are empty d1cf75e48a552efcf10d913c39373cb2eb025b34